Index: src/kuickshow.desktop
===================================================================
--- src/kuickshow.desktop
+++ src/kuickshow.desktop	2009-12-26 17:33:04.000000000 +0100
@@ -41,7 +41,7 @@
 Name[x-test]=xxKuickshowxx
 Name[zh_CN]=Kuickshow
 Name[zh_TW]=Kuickshow
-Exec=kuickshow -caption "%c" %U
+Exec=XLIB_SKIP_ARGB_VISUALS=1 kuickshow -caption "%c" %U
 Icon=kuickshow
 Type=Application
 # TODO: What is image/x-eim? Register it to freedesktop.org if you need it
